packages/flutter/lib/src/material/shaders/ink_sparkle.frag

```diff
@@ -0,0 +1,102 @@
+#version 320 es
```

Comment:
This should have a copyright header, right?

Reply:
Ideally, we also extend the license header check script that warns if a file of this type is added without copyright header.